.github/workflows/indent.yml

```diff
@@ -0,0 +1,27 @@
+name: indent
+
+on: [push, pull_request]
+
+concurrency:
+  group: ${ {github.event_name }}-${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{github.event_name == 'pull_request'}}
+
+jobs:
+  indent:
+    # run the indent checks
+
+    name: indent
+    runs-on: [ubuntu-20.04]
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 100
+    - name: setup
+      run: |
+        sudo apt-get install --yes snapd
+        sudo snap install shfmt
```

Comment:
This won't give you the right version in the future.

Reply:
Fixed to use the current stable release version `--channel=latest/stable`. The version check tests now for the current version 3.3.1 or any later version.